minato_namikaze/discordbot.py

```diff
@@ -74,11 +80,9 @@ def get_prefix(self, bot, message):
     
     def run(self):
         try:
-            import sentry_sdk
-            from sentry_sdk.integrations import AioHttpIntegration
 
             sentry_sdk.init(
-                sentry_link,
+                "https://mydsn@sentry.io/123",
```

Comment:
change `"https://mydsn@sentry.io/123"` to `sentry_link` since it will be directly coming from `vars.py`

Reply:
ok so i will change it to 
```py
Tokens.sentry_link.value,
``` 
tho it is throwing another error:
```
Unsupported scheme ''
``` 
idk why.. so do you've any idea on this or shall i just push it?

Reply:
well then just push it, I will see what to do , when I get time :)

Reply:
alright